app/views/admin/settings/forms/_emails.html.erb

```diff
@@ -1,22 +1,38 @@
-<div class="card mt-3">
-  <%= render partial: "admin/shared/card_header",
-             locals: {
-               header: "Emails",
-               state: "collapse",
-               target: "emailBodyContainer",
-               expanded: false
-             } %>
-  <div id="emailBodyContainer" class="card-body collapse hide" aria-labelledby="emailBodyContainer">
-    <fieldset class="grid gap-4">
-      <div class="crayons-field">
-        <label for="forem_instance_email" class="crayons-field__label">
-          Default
-        </label>
-        <div class="crayons-field__description">
-          Default (set via <span class="ff-monospace">default_email</span> environment variable)
+<%= form_for(Settings::General.new,
```

Comment:
The emails section changed in the following way: 

- I removed the `default_email` field from the `Emails` section since it was not adding any value since we cannot change it. 
- In order to still alert users about the default email, I updated the SMTP (Email Server Section) section in a separate [PR](https://github.com/forem/forem/pull/16223)  with a description that outlined that no-reply@forem.com will be used as their default email if they are a Forem Cloud Client. They can override this with custom SMTP settings.
- In the Emails Section I grouped the  contact field and the Periodic email digest field.